DifferentialGrowth2D/DifferentialGrowth2D.js

```diff
@@ -0,0 +1,61 @@
+import { Rectangle } from "./rectangle.js";
+import { Quadtree } from "./quadtree.js";
+import { DifferentialPolyline } from "./DifferentialPolyline.js";
+
+const WIDTH = 500;
+const HEIGHT = 700;
+const BOUNDS = new Rectangle(0, 0, WIDTH, HEIGHT);
+const QUADTREE = new Quadtree(BOUNDS);
+
+const points = [];
+const points2 = [];
+const INITIAL_POINTS = 20;
+const OFFSET = HEIGHT / 12;
+for (let i = 0; i < INITIAL_POINTS; i++) {
+  const angle = (2.0 * Math.PI * i) / INITIAL_POINTS;
+  const x = WIDTH / 2 + 50 * Math.cos(angle);
+  const y = HEIGHT / 2 + 50 * Math.sin(angle);
+  points.push([x, y - OFFSET]);
+  points2.push([x, y + OFFSET]);
```

Comment:
Why do I even bother making arrays only to turn them into `Vector`? 

except turn these into `Point`

Reply:
For now, make these Vectors, that way it'll be easier to search and replace later